topics/objective-c/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Tom Love and Brad Cox
+display_name: Objective-C
+logo: objective-c.png
+related: language
+released: '1984'
+short_description: Objective-c is general purpose, object oriented programming language used for mac and iOS operating system. 
```

Comment:
- `Objective-c` to `Objective-C`
- `object oriented` to ` object-oriented`
- `mac` to `macOS`
- `operating system` to `operating systems`